src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -28,21 +45,35 @@ public Arm() {
       */
     }
 	
-    public void setArmPos() {
+    public void setArmGoal(double targetPosition, double targetVelocity) {
       //Sets arm to the optimal angle for amp, speaker and Ground intake | used to score in amp
       //these values are in constants
-      //pass in where scorign and use switch statement
-    
-
+      //pass in where scorign and use switch statement to alternate between each angle needed
```

Comment:
What exactly is the switch statement for? Aren't you just going to set the goal to where you want?